plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/client/PinotClient.java

```diff
@@ -228,8 +240,15 @@ private <T> T sendHttpGetToBrokerJson(String table, String path, JsonCodec<T> co
     {
         ImmutableMultimap.Builder<String, String> additionalHeadersBuilder = ImmutableMultimap.builder();
         brokerAuthenticationProvider.getAuthenticationToken().ifPresent(token -> additionalHeadersBuilder.put(AUTHORIZATION, token));
+        final HttpUriBuilder builder = useProxy ?
+                uriBuilderFrom(getControllerUrl()) :
```

Comment:
If `useProxy` is true, use `getControllerUrl` sounds weird. Could you rename the property and relevant variables? Also, can I what's the situation/benefit of this change?

Reply:
Just change to proxy-rest-uri, and if this value is set, use it.